Dockerfile

```diff
@@ -30,7 +30,7 @@ COPY --from=tinygo-base /tinygo/targets /tinygo/targets
 RUN cd /tinygo/ && \
     apt-get update && \
     apt-get install -y make clang-11 libllvm11 lld-11 && \
-    make wasi-libc
+    make wasi-libc binaryen
```

Comment:
Oops, just noticed this. Never mind my other question.

Reply:
I just double checked and this doesn't actually work anymore, so I do actually need to fix it.